backend/api/tests/views/test_views_compute_plan_graph.py

```diff
@@ -236,9 +236,7 @@ def test_n_plus_one_queries_compute_graph(authenticated_client):
     url = reverse("api:workflow_graph", args=[compute_plan.key])
 
     with utils.CaptureQueriesContext(connection) as queries:
-        authenticated_client.get(
-            url, {"HTTP_SUBSTRA_CHANNEL_NAME": "mychannel", "HTTP_ACCEPT": "application/json;version=0.0"}
-        )
+        authenticated_client.get(url, {"HTTP_ACCEPT": "application/json;version=0.0"})
```

Comment:
```suggestion
        authenticated_client.get(url, **self.extra)
```

Reply:
this isn't in the class ComputePlanGraphViewTests thus self.extra does not exist.
Is this fixture suppose to be in the class ?